sbt-crossproject/src/main/scala/sbtcrossproject/CrossProject.scala

```diff
@@ -2,6 +2,8 @@ package sbtcrossproject
 
 import sbtcrossproject._
 
+import scala.collection.immutable.HashMap
+import scala.collection.immutable.ListSet
 import scala.language.implicitConversions
```

Comment:
```suggestion
import scala.language.implicitConversions

import scala.collection.immutable.HashMap
import scala.collection.immutable.ListSet
```
Language imports should always come first, in a dedicated "paragraph".